modules/comment/comment.resolver.js

```diff
@@ -16,6 +16,19 @@ const commentsQuery = {
     }
   },
 
+  getRecentComments: async (parent, args) => {
+    try {
+      return commentsService.getRecentComments(args.comentDate);
+    } catch (error) {
+      return [
```

Comment:
we need to return RuleError

Reply:
in RuleError please use error.message and error.statusCode